ts/Grid/Core/Grid.ts

```diff
@@ -800,15 +801,13 @@ class Grid {
 
         this.renderDescription();
 
-        if (this.options?.credits?.enabled) {
-            this.credits = new Credits(this);
-        }
-
         this.accessibility?.setA11yOptions();
 
         if (this.viewport?.virtualRows) {
             this.viewport.reflow();
         }
+
+        fireEvent(this, 'afterRenderViewport');
```

Comment:
I hope that the order is not so important here and that it does not generate problems if the credits init is after reflow and a11y initialization.

Reply:
I did not notice.